tools/harness-thci/OpenThread.py

```diff
@@ -1888,22 +1888,18 @@ def setNetworkIDTimeout(self, iNwkIDTimeOut):
 
     @API
     def setKeepAliveTimeOut(self, iTimeOut):
-        """set keep alive timeout for device
-           has been deprecated and also set SED polling rate
+        """set child timeout for device
 
         Args:
-            iTimeOut: data poll period for sleepy end device
+            iTimeOut: child timeout for device
 
         Returns:
-            True: successful to set the data poll period for SED
-            False: fail to set the data poll period for SED
+            True: successful to set the child timeout for device
+            False: fail to set the child timeout for device
         """
         print('%s call setKeepAliveTimeOut' % self)
```

Comment:
also remove this line since `@API` is already printing verbosely

Reply:
Sure, will do.